cpu/cc2538/radio/cc2538_rf_netdev.c

```diff
@@ -329,19 +335,19 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
 
     rfcore_read_fifo(buf, pkt_len);
 
+    int8_t rssi_val = rfcore_read_byte() + CC2538_RSSI_OFFSET;
+    uint8_t crc_corr_val = rfcore_read_byte();
+
     if (info != NULL && RFCORE->XREG_RSSISTATbits.RSSI_VALID) {
-        uint8_t corr_val;
-        int8_t rssi_val;
         netdev_ieee802154_rx_info_t *radio_info = info;
-        rssi_val = rfcore_read_byte() + CC2538_RSSI_OFFSET;
 
         RFCORE_ASSERT(rssi_val > CC2538_RF_SENSITIVITY);
 
         /* The number of dB above maximum sensitivity detected for the
          * received packet */
         radio_info->rssi = -CC2538_RF_SENSITIVITY + rssi_val;
 
-        corr_val = rfcore_read_byte() & CC2538_CORR_VAL_MASK;
+        uint8_t corr_val = crc_corr_val & CC2538_CORR_VAL_MASK;
```

Comment:
These changes are valid but not related to the commit (2eb2ac2bbdf606a4d29e3f97415b7a0a0cf0cb5f) they are in, can you move them to 84860e5e717c4695b4561e3e077dd0c50b2d5615.

Reply:
The reason these changed are in this commit is to ensure that the whole frame in `RX FIFO` is consumed

Reply:
Ohhh got it I'm sorry.